src/popup/components/TransferSendForm.vue

```diff
@@ -62,6 +63,41 @@
       </template>
     </InputAmount>
 
+    <template v-if="showPayloadForm">
+      <div
+        v-if="!(formModel.payload && formModel.payload.length)"
+        class="payload-add-wrapper"
+      >
+        <div
+          class="payload-btn"
+          @click="editPayload"
+        >
+          <PayloadAddIcon class="payload-add-icon" />
+          PAYLOAD
+        </div>
+        <QuestionIcon
+          class="icon-btn"
+          @click="openPayloadInformation"
+        />
```

Comment:
IconBtn, then you'll be able to remove styles for it

Reply:
this is totally other style and hover